### ISSUE TITLE
repeated error: Method wrapper not found on class stdgo.internal.Macro

### ISSUE THREAD

**Issue by elliott5** [go2hx/go2hx#113]
23 of the [gobyexample.com ](https://gobyexample.com) short programs fail with errors including `Method wrapper not found on class stdgo.internal.Macro`.

**Comment by PXshadow** [comment]
Is this still an issue, and what is the name of the test @elliott5 ?

**Comment by elliott5** [comment]
I'll see if I can find the test code.

**Comment by elliott5** [comment]
This is no-longer a repeated error.

Results of [gobyexample.com ](https://gobyexample.com)code testing below:
```
======= SUMMARY RESULTS ======

--> Correct:  29.3%
arrays.go atomic-counters.go channel-directions.go channel-synchronization.go channels.go closures.go errors.go for.go functions.go hello-world.go if-else.go methods.go multiple-return-values.go range.go recover.go recursion.go sorting-by-functions.go sorting.go string-functions.go struct-embedding.go variables.go variadic-functions.go

--> Executed go2hx successfully, but output != go:  30.7%
channel-buffering.go closing-channels.go command-line-flags.go constants.go epoch.go goroutines.go interfaces.go maps.go mutexes.go pointers.go random-numbers.go range-over-channels.go select.go stateful-goroutines.go strings-and-runes.go structs.go switch.go tickers.go time-formatting-parsing.go time.go values.go waitgroups.go worker-pools.go

--> Excluded, either yet to be implemented or unsupported functionality:   6.7%
non-blocking-channel-operations.go number-parsing.go rate-limiting.go signals.go timers.go

--> go2hx compilation error:  33.3%
base64-encoding.go context.go defer.go directories.go embed-directive.go environment-variables.go execing-processes.go file-paths.go generics.go http-clients.go http-servers.go json.go line-filters.go panic.go reading-files.go regular-expressions.go sha256-hashes.go slices.go spawning-processes.go temporary-files-and-directories.go text-templates.go timeouts.go url-parsing.go writing-files.go xml.go

--> haxelib error:   0.0%


started: Mon Jan 23 17:12:49 GMT 2023  ended: Mon Jan 23 17:12:49 GMT 2023  duration: 8m39.637233792s
```
